e2e/fixtures/base/src/routes/index.tsx

```diff
@@ -0,0 +1,48 @@
+import type { JSX } from 'react'
+import type { TuonoProps } from 'tuono'
+
+interface IndexProps {
+  subtitle: string
+}
+
+export default function IndexPage({
+  data,
+  isLoading,
+}: TuonoProps<IndexProps>): JSX.Element {
+  if (isLoading) {
+    return <h1>Loading...</h1>
+  }
+
+  return (
+    <>
+      <header className="header">
+        <a href="https://crates.io/crates/tuono" target="_blank">
+          Crates
+        </a>
+        <a href="https://www.npmjs.com/package/tuono" target="_blank">
+          Npm
+        </a>
+      </header>
```

Comment:
I don't think we need this part and all associated CSS

```suggestion
```

Reply:
Removed 👍